backend/infrahub/graphql/app.py

```diff
@@ -185,7 +184,7 @@ async def _handle_http_request(
     ) -> JSONResponse:
         if request.app.state.response_delay:
             self.logger.info(f"Adding response delay of {request.app.state.response_delay} seconds")
-            time.sleep(request.app.state.response_delay)
```

Comment:
This was something that @fatih-acar added to test some conditions where the app was completely blocked. I don't know if this section is still relevant to keep?

Reply:
I kept it, just moved it to an `await asyncio.sleep()` one. Would it be OK like so?

Reply:
using asyncio.sleep will block the current async coroutine, not the whole app, that's why having time.sleep is useful here (we want to fully block the app to simulate execution lag/slowness, not sure if we will get the same behavior using asyncio.sleep)